Assets/Python/GreatPeople.py

```diff
@@ -2341,10 +2374,12 @@ def setup():
 		"fJeanne d'Arc", # 15th
 		iRenaissance,
 		u"Louis de Bourbon-Condé", # 17th
-		"Henri de la Tour d'Auvergne", # 17th
```

Comment:
Henri de Turenne

Reply:
I think the nickname alone is better, since I don't think he is really referred to as "Henri de Turenne" at all. This is similar to Molière, Voltaire, Montesquieu, all of whom are much better known by their nickname/pseudonym than their full name.